envbuilder.go

```diff
@@ -394,6 +396,14 @@ func Run(ctx context.Context, options Options) error {
 		})
 	}
 
+	// temp move of all ro mounts
+	tempRemountDest := filepath.Join("/", MagicDir)
+	ignorePrefixes := []string{tempRemountDest, "/proc", "/sys"}
+	remount, err := ebutil.TempRemount(options.Logger, tempRemountDest, ignorePrefixes...)
+	if err != nil {
+		return fmt.Errorf("temp remount: %w", err)
+	}
```

Comment:
I was going to suggest a defer here, but perhaps it's not so important that we restore the mounts on error.

Reply:
I can put the logic inside a `sync.Once` and then we can just `defer restore()`